templates/sleep/TODO

```diff
@@ -0,0 +1,4 @@
+- skip power states not in /sys/power/state
+- probably loop a few times (at least two)
+- may need to configure RTCs other than rtc0
+- find a way to not replicate the actual test cases in each template
```

Comment:
You can (and probably should) already fix that with a separate template fragment, say `templates/sleep/sleep.jinja2` which will contain the test definition.  Then you can include that in each template variant with the jinja2 `include` statement.

Reply:
Done, thanks.